src/cljs/main/broadfcui/page/profile.cljs

```diff
@@ -38,9 +38,9 @@
 (defn get-url-search-param [param-name]
   (.get (js/URLSearchParams. js/window.location.search) param-name))
 
-(defn auth-url-link [href text]
+(defn auth-url-link [href text provider]
   (if href
-    (links/create-external {:href href :target "_self"} text)
+    (links/create-external {:href href :target "_self" :data-test-id provider :id "provider-link"} text)
```

Comment:
also, `:id "provider-link` - `id` attributes should be unique on a page.  I think you can change `:id` to `:class` here (and update your test code too) and you should get the same effect.